### PR TITLE
Travis: compile with openssl on osx [WIP]

### PR DESCRIPTION
**Pull request by ctrlaltca**
Changes:
 * Use homebrew to install openssl
 * add cmake flag to compile with openssl support

Work in progress: I have to check Travis build log to ensure everything is Ok before merging.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -85,9 +85,10 @@ install:
     - |
         if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
           brew update
-          brew install qt5 enchant
+          brew install qt5 enchant openssl
           # want cmake newer than by default in travis osx
           brew upgrade cmake
+          CMAKE_ARGS="$CMAKE_ARGS -DOPENSSL_ROOT_DIR=$(brew --prefix qt5)"
```
Comment (DarthGandalf) [inline .travis.yml]: probably should be openssl instead of qt5

**Comment by Stanzilla** [comment]
build failure is because of https://github.com/kvirc/KVIrc/issues/2446

**Comment by Stanzilla** [comment]
@ctrlaltca this good now?

**Comment by ctrlaltca** [comment]
From the build logs:
```
-- Found OpenSSL: /usr/local/opt/openssl/lib/libcrypto.dylib (found version "1.0.2s") 
...
-- Cryptography:
--    Crypt support and engines   : Yes
--    OpenSSL support             : Yes
...
-- fixup_bundle: copying...
...
-- 4/186: copying '/usr/local/Cellar/openssl/1.0.2s/lib/libcrypto.1.0.0.dylib'
...
-- 13/186: copying '/usr/local/opt/openssl/lib/libssl.1.0.0.dylib'
```
Everything looks fine

**Pull request merged**